docs/design/src/lib/MutexTransitionWrapper.c

```diff
@@ -2,7 +2,7 @@
 #include "mcmini/spy/intercept/interception.h"
 #include "mcmini/shared_sem.h"
 #include "mcmini/shared_transition.h"
-#include "wrappers.h"
+#include "MutexTransiitionWrappers.h"
```

Comment:
The include path should be fully written out:
```cpp
#include "mcmini/spy/intercep/MutexTransitionWrappers"
```
and again the file naming as above.